### ISSUE TITLE
center of two finger zoom should be between the two fingers 

### ISSUE THREAD

**Issue by HoWilgh** [SimpleMobileTools/Simple-Draw#138]
Currently the two fingers zoom uses the center of the canvas as origin for the zoom. 
This dramatically limits the use cases of the zoom functionality. 

Usually, the zoom gesture users the middle of the distance between the two fingers as origin. This allows to navigate in the drawing and zoom a specific region of the drawing to e.g. improve it.

**Comment by CactiChameleon9** [comment]
Sorry to ask, but is this supposed to be implemented? I cannot seem to reproduce the feature. When I zoom, I always zoom in on what would be the actual center of the image would be - not the center of where my screen showing or between my two fingers.

**Comment by tibbi** [comment]
what app version are you using?

**Comment by CactiChameleon9** [comment]
6.8.0 Pro from f-droid

**Comment by tibbi** [comment]
right, it works as expected now though, you can move the canvas with gestures too

**Comment by CactiChameleon9** [comment]
So it is expected that if I:
1. Zoom out
2. Move the canvas
3. Draw a bit so you can tell
4. Zoom in
I end up zooming towards what is off the screen (the centre)?

**Comment by tibbi** [comment]
you can zoom and move with 1 gesture, it is really easy and intuitive

**Comment by CactiChameleon9** [comment]
So yes its intended, and the solution/workaround is move your fingers a vairable amount depending on how far away you are from the center while zooming?
